docs/management/connectors/action-types/index.asciidoc

```diff
@@ -46,10 +45,8 @@ xpack.actions.preconfigured:
     actionTypeId: .index
     config:
       index: .kibana
-      refresh: true <1>
```

Comment:
@lcawl I also remove this from the preconfigured connector config because we want to discourage people from setting the value to `true`. When not set, it will default to `false`